sdk/monitor/monitor-opentelemetry-exporter/test/internal/logUtils.spec.ts

```diff
@@ -38,18 +39,18 @@ function assertEnvelope(
   expectedBaseData?: Partial<MonitorDomain>,
   expectedTime?: Date,
 ): void {
-  assert.ok(envelope);
-  assert.strictEqual(envelope.name, name);
-  assert.strictEqual(envelope.sampleRate, sampleRate);
-  assert.deepStrictEqual(envelope.data?.baseType, baseType);
+  assert.isDefined(envelope);
```

Comment:
would `assert.exist` give you better typing so you don't need the `?` below? Sometimes it works for me and sometimes it does not - I think it depends on where the typings come from (chai or vitest's chai) but one of them is a typeguard 

this is a test, so not too concerned

Reply:
Yes, that would be ideal, and noted for the next go around.